examples/3d-tiles/tileset-3d-layer.js

```diff
@@ -165,34 +173,72 @@ export default class Tileset3DLayer extends CompositeLayer {
     });
   }
 
-  _resolveCoordinateProps(tileHeader) {
+  /* eslint-disable-next-line complexity */
+  _resolveTransformProps(tileHeader) {
     if (!tileHeader || !tileHeader.content) {
       return {};
     }
 
-    const {coordinateSystem, coordinateOrigin} = this.props;
+    const {coordinateSystem, coordinateOrigin, isWGS84} = this.props;
     const {rtcCenter} = tileHeader.content;
+    const {transform} = tileHeader.userData;
 
-    const coordinateProps = {
-      coordinateSystem: coordinateSystem || COORDINATE_SYSTEM.LNGLAT,
-      coordinateOrigin
-    };
+    const transformProps = {};
+
+    if (coordinateSystem) {
+      transformProps.coordinateSystem = coordinateSystem;
+    }
+    if (coordinateOrigin) {
+      transformProps.coordinateOrigin = coordinateOrigin;
+    }
+
+    if (isWGS84) {
+      transformProps.coordinateSystem = transformProps.coordinateSystem || COORDINATE_SYSTEM.LNGLAT;
+      return transformProps;
```

Comment:
Why return here? If WGS84, don't we need to continue and extract transform and rtcCenter?